docker/nginx-templates/default.nginx.conf.template

```diff
@@ -0,0 +1,30 @@
+server {
+    listen 80;
+
+    root /usr/share/nginx/html;
+
+    sendfile on;
+    tcp_nopush on;
+
+    gzip on;
+    gzip_types text/plain text/css application/javascript application/json image/svg+xml;
+
+#    location /static/ {
+#        alias /.fs/static/;
+#        expires 365d;
+#    }
```

Comment:
Since we've added nginx into the docker stack, I suggest adding this as well since it's fairly easily achieved by just adding a volume for static files folder in docker compose. Configured in `STATIC_ROOT` and generated with the `collectstatic` command.